instruments/keithley/keithley485.py

```diff
@@ -334,7 +334,7 @@ def _get_status_word(self):
             statusword = self.query("U0X")
             tries -= 1
 
-        if statusword is None:
+        if tries == 0:
```

Comment:
Above, `statusword = ""` is introduced. Then instrument reads are tried several times. `statusword` can never be `None`. An `if statusword` comparison alone furthermore would not do the job in case the instrument returns something invalid. The while loop would still continue in this case until it runs out of tries, and the statusword returned would be bogus. 
Checking for `tries==0` avoids this issue.